tools/clang/test/CodeGenSPIRV/semantic.stencil-ref.ps.hlsl

```diff
@@ -0,0 +1,15 @@
+// Run: %dxc -T ps_6_0 -E main
+
+// CHECK: OpCapability StencilExportEXT
+
+// CHECK: OpExtension "SPV_EXT_shader_stencil_export"
```

Comment:
According to https://www.khronos.org/registry/spir-v/extensions/EXT/SPV_EXT_shader_stencil_export.html
It seems that you must also add `OpExecutionMode ExecutionModeStencilRefReplacingEXT` ?

Reply:
Glslang has a GLSL test which also uses this extension builtin: https://github.com/KhronosGroup/glslang/blob/master/Test/baseResults/spv.shaderStencilExport.frag.out. It does not add the execution mode.

I've filed an issue against glslang. Will fix this if the outcome is we need it.